.github/workflows/pypi-publish.yml

```
@@ -0,0 +1,35 @@
+# Upload a Python package when a release is created
+# https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows
+
+name: Publish Python 🐍 distributions 📦 to PyPI
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  build-n-publish:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write # for trusted publishing
```

```suggestion
    environment:
      name: pypi
      url: https://pypi.org/p/fedora-distro-aliases
    permissions:
      id-token: write # for trusted publishing
```
`pypa/gh-action-pypi-publish` recommends to include an environment. Not sure what the technical reasons for this is, but why not go with their recommendation